qutebrowser/browser/webengine/webenginetab.py

```diff
@@ -79,6 +83,47 @@ def init():
 }
 
 
+def inject_userscripts():
```

Comment:
Since this isn't actually related to the tab at all, I think it should go to `webenginesettings.py`